doc/howto/custom_field_type.md

```diff
@@ -160,6 +160,31 @@ class RelationFieldDefinitionMapper extends DecoratingFieldDefinitionMapper impl
 }
 ```
 
+#### Field Definition Input Mappers
+As of v1.0.4, an extra interface is available for mutation input type handling, `FieldDefinitionInputMapper`.
+It is used if the input for this field depends on the field definition. For instance, `ezmatrix`
+generates its own input types depending on the configured columns.  It defines an extra method, `mapToFieldValueInputType`, 
+that returns a GraphQL type for a Field Definition.
+
+Example:
+```
+class MyFieldDefinitionMapper extends BaseFieldDefinitionMapper implements FieldDefinitionMapper, FieldDefinitionInputMapper
+{
+    public function mapToFieldValueInputType(ContentType $contentType, FieldDefinition $fieldDefinition): ?string
+    {
+        // Forward compatibility with 2.x.
+        // It will only call the parent's method if it exists.
+        if (!$this->canMap($fieldDefinition) && is_callable('parent::mapToFieldValueInputType')) {
```

Comment:
Interesting forward compatibility approach :)

In this case I'd rather go for implementing `mapToFieldValueInputType` in `BaseFieldDefinitionMapper` directly, so you can skip ` && is_callable('parent::mapToFieldValueInputType')` in this example.

But don't have a strong opinion here :)

Reply:
I like the idea. This check was acceptable for forward compatibility I think, but the abstract implementation seems to make more sense. I'll give it a try, thank you !

Reply:
Done in dec6ddf4ac5c3eb7397f67cfc34ba9e2ea620be4.